core/domain/suggestion_jobs_one_off_test.py

```diff
@@ -44,6 +44,137 @@
+    QUESTION_ID = 'question_id'
+
+    def setUp(self):
+        super(QuestionSuggestionMigrationJobManagerTests, self).setUp()
+
+        self.signup(self.ALBERT_EMAIL, self.ALBERT_NAME)
+        self.albert_id = self.get_user_id_from_email(self.ALBERT_EMAIL)
+        self.process_and_flush_pending_mapreduce_tasks()
+        self.skill_id = 'skill_id'
+        self.save_new_skill(
+            self.skill_id, self.albert_id, description='Skill Description')
+
+        suggestion_change = {
+            'cmd': (
+                question_domain
+                .CMD_CREATE_NEW_FULLY_SPECIFIED_QUESTION),
+            'question_dict': {
+                'question_state_data': self._create_valid_question_data(
+                    'default_state').to_dict(),
+                'language_code': 'en',
+                'question_state_data_schema_version': (
+                    feconf.CURRENT_STATE_SCHEMA_VERSION),
+                'linked_skill_ids': [self.skill_id],
+                'inapplicable_skill_misconception_ids': []
+            },
+            'skill_id': self.skill_id,
+            'skill_difficulty': 0.3
+        }
+
+        self.suggestion = suggestion_services.create_suggestion(
+            suggestion_models.SUGGESTION_TYPE_ADD_QUESTION,
+            suggestion_models.TARGET_TYPE_SKILL, self.skill_id, 1,
+            self.albert_id, suggestion_change, 'test description')
+
+    def _run_job_and_verify_output(self, expected_output):
+        """Runs the QuestionSuggestionMigrationJobManager and
+        verifies that the output matches the expected output.
+
+        Args:
+            expected_output: list(str). The expected output from the one off
+                job.
+        """
+        job_id = (
+            suggestion_jobs_one_off
+            .QuestionSuggestionMigrationJobManager.create_new())
+        (
+            suggestion_jobs_one_off
+            .QuestionSuggestionMigrationJobManager.enqueue(job_id)
+        )
+        self.process_and_flush_pending_mapreduce_tasks()
+
+        actual_output = (
+            suggestion_jobs_one_off
+            .QuestionSuggestionMigrationJobManager.get_output(job_id))
+
+        self.assertItemsEqual(actual_output, expected_output)
+
+    def test_migration_job_does_not_convert_up_to_date_suggestion(self):
+        self.assertEqual(
+            self.suggestion.change.question_dict[
+                'question_state_data_schema_version'],
+            feconf.CURRENT_STATE_SCHEMA_VERSION)
+
+        expected_output = [u'[u\'SUCCESS\', 1]']
+        self._run_job_and_verify_output(expected_output)
+
+        updated_suggestion = (
+            suggestion_services.get_suggestion_by_id(
+                self.suggestion.suggestion_id))
+        self.assertEqual(
+            updated_suggestion.change.question_dict[
+                'question_state_data_schema_version'],
+            feconf.CURRENT_STATE_SCHEMA_VERSION)
+
+    def test_migration_job_skips_deleted_question_suggestion(self):
```

Comment:
Do we need this test? (As the suggestion models get completely delete so we aren't skipping any models.)

Either remove the test or rename the test function name.

Reply:
Renamed the function* DOne!